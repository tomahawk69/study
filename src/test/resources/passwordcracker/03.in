2
3
la fa ma
lama
3
la fa ma
lamala
