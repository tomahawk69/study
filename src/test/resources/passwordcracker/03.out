la ma
la ma la